entrypoint.sh

```diff
@@ -7,13 +7,14 @@ export VAULT_SKIP_VERIFY=true
 export CONSUL_HTTP_ADDR=$CONSUL_ADDR:8500
 export SERVICE_ACCOUNT_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
 
-export VAULT_PATH=auth/kubernetes/login
-if [ $KUBERNETES_SERVICE_HOST == '172.20.0.1' ] || [ $KUBERNETES_SERVICE_HOST == '10.100.0.1' ]
-then
-  export VAULT_PATH=auth/eks/login
-fi
+export VAULT_PATH=auth/eks/login
+
+# TODO: Read the vault path from service in future
+SERVICE_URL="vault-path.infra.svc.cluster.local"
```

Comment:
Add a comment describing a sample value. It would be easier if we need to play with this in future then.